packages/mco/components/disaster-recovery/applications/failover-and-relocate-modal/peer-status.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import { getName } from '@odf/shared/selectors';
+import {
+  StatusIconAndText,
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { TFunction } from 'i18next';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import { UnknownIcon } from '@patternfly/react-icons';
+import { PEER_READINESS } from '../../../../constants';
+import { findPeerStatus } from '../../../../utils';
+import {
+  ACTION_TYPE,
+  FailoverAndRelocateState,
+  FailoverAndRelocateAction,
+  FailoverAndRelocateType,
+} from './reducer';
+
+const initalPeerStatus = (t: TFunction) => ({
+  peerReadiness: {
+    icon: <UnknownIcon />,
+    text: PEER_READINESS(t).UNKNOWN,
+  },
+  lastHealthCheck: {
+    icon: <UnknownIcon />,
+    text: 'Unknown',
+  },
+  lastSnapshotTakenOn: {
+    icon: <UnknownIcon />,
+    text: 'Unknown',
+  },
+});
```

Comment:
can't we do this, since all 3 initial statuses are same ?
```suggestion
const initalPeerStatus = (t: TFunction) => ({
  initialStatus: {
    icon: <UnknownIcon />,
    text: t('ns~Unknown'),
  },
});
```